### PR TITLE
fixes 'Move components in core into a GCP specific ksonnet package'

### LINKED ISSUE #1996: Move components in core into a GCP specific ksonnet package
See #42 

We should move the components in the core package that are GCP specific into their own ksonnet package.

/cc @kunmingg

### PR DESCRIPTION
**Pull request by kkasravi**
fixes #1996

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2043)

### CODE REVIEW

File: kubeflow/gcp/OWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+approvers:
+  - gaocegege
```
Comment (jlewi) [inline kubeflow/gcp/OWNERS]: Not the right set of approvers for GCP :)
Please list Googlers
jlewi, r2d4,  richardsliu, kunming, llunnn, abhi-g

**Comment by jlewi** [comment]
Thanks so much. For doing this @kkasravi 

Test is likely failing because you need to update kfctl

File: kubeflow/gcp/OWNERS (hunk 1 above)
Comment (kkasravi) [reply kubeflow/gcp/OWNERS]: my bad, i didn't realize that's where you look

File: kubeflow/gcp/OWNERS [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+reviewers:
```
Comment (jlewi) [inline kubeflow/gcp/OWNERS]: If there are new reviewers just remove the reviewers section.

**Comment by jlewi** [comment]
looks like the OWNERs file might still need to be updated.

**Comment by kkasravi** [comment]
@jlewi OWNERs file should be good now

File: kubeflow/gcp/OWNERS (hunk 1 above)
Comment (jlewi) [reply kubeflow/gcp/OWNERS]: Can you remove gaocegege please?

**Comment by jlewi** [comment]
@kkasravi Can you remove gaocegege as an approver since this is GCP specific?

**Pull request merged**